cocos/3d/skeletal-animation/skeletal-animation.ts

```diff
@@ -185,6 +185,16 @@ export class SkeletalAnimation extends Animation {
         this._removeAllUsers();
```

Comment:
Shouldn't `onDestroy` stop all playing states and remove from the animation manager

Reply:
Yes it does: https://github.com/cocos/cocos-engine/blob/develop/cocos/3d/skeletal-animation/skeletal-animation.ts#L183 , https://github.com/cocos/cocos-engine/blob/develop/cocos/core/animation/animation-component.ts#L197